unlock-app/src/components/interface/connect/EmailAccount.tsx

```diff
@@ -181,6 +186,8 @@ export interface SignWithGoogleProps {
 
 const SignWithGoogle = ({ callbackUrl, isSignUp }: SignWithGoogleProps) => {
   const signWithGoogle = () => {
+    localStorage.setItem('nextAuthProvider', 'google')
```

Comment:
Ca you please do tht in a consistent place with the other place where we set this value?